src/ErrorBoundary.ts

```diff
@@ -18,7 +16,7 @@ const initialState: ErrorBoundaryState = {
 };
 
 export class ErrorBoundary extends Component<
-  PropsWithRef<PropsWithChildren<ErrorBoundaryProps>>,
```

Comment:
> 1. I'm curious whether PropsWithRef is necessary.

I don't think this is a great reason to remove a type.

I'm also not certain whether that meta type is necessary. It was added by @kentcdodds a few years ago when migrating this project to TypeScript (#76)

Reply:
Okay, I added commit to leave PropsWithRef in place for now. [dca7aa7](https://github.com/bvaughn/react-error-boundary/pull/152/commits/dca7aa7bb530d3cc9a350e7d581c58f157f72d69)

@kentcdodds, Could you explain if there is reason to leave PropsWithRef? if it's not necessary, I want to remove it too.

Reply:
I'm afraid it's now been long enough that I couldn't say whether that type is very necessary. This was also long enough ago that there could be a better way to do whatever I was trying to do these days 😅

Reply:
That's all too familiar 😄 Appreciate you dropping by to say hi though